plugins/backup/networker/src/main/java/org/apache/cloudstack/backup/NetworkerBackupProvider.java

```diff
@@ -615,6 +611,7 @@ public Backup createNewBackupEntryForRestorePoint(Backup.RestorePoint restorePoi
             backup.setAccountId(vm.getAccountId());
             backup.setDomainId(vm.getDomainId());
             backup.setZoneId(vm.getDataCenterId());
+            backup.setName(vm.getHostName() + '-' + new SimpleDateFormat("yyyy-MM-dd'T'HH:mmX").format(new Date()));
```

Comment:
This is a repeating code. Would it make sense to add method in BackupProvider interface or some other common class to get the backup name?